docs/protobuf_getting_started.md

```diff
@@ -13,11 +13,14 @@ This document provides some background as to how to generate a Protobuf definiti
 
 ## Understanding the Output Protobuf Files
 
-### Filesystem Hierarchy
+The `ygot` package contains a `proto_generator` binary ([source](https://github.com/openconfig/ygot/tree/master/proto_generator)). This binary is used to generate the protocol buffer definitions. Rather than outputting an individual protobuf file with all message definitions within it, this binary has two modes of output:
 
-The `ygot` package contains a `proto_generator` binary ([source](https://github.com/openconfig/ygot/tree/master/proto_generator)). This binary is used to generate the protocol buffer definitions. Rather than outputting an individual protobuf file with all message definitions within it, the generate outputs a hierarchy of files following the schema of the YANG module that is supplied.
+  * **Nested Messages**: In this output mode, one protobuf package per YANG module is output.
+  * **Hierarchical Packages**: In this case, each level of the YANG schema tree hierarchy has an individual protobuf package output for it.
 
-For example,  with the following YANG module:
+The default output mode is to produce nested messages. The `-nested_messages` flag of the `proto_generator` binary can be set to false to produce hierarchical packages.
```

Comment:
In general I think it's better because most people think just including a boolean flag makes it true i.e. "--recurse" would be true by default if it's included and false if omitted - see https://gobyexample.com/command-line-flags for a similar example.
But it's probably case by case? i.e. if it means you have to call a flag something silly like "dontRecurse" it's probably better to leave it to true by default. My 2c :-)

Reply:
Done -- the flag changed here, but I left the switch in the library being the other way around since this is a relatively huge change through that code. I think it's OK to have these two defaults separate.